packages/RichTextEditing-Core.package/TextStructure.class/instance/attributes..st

```diff
@@ -0,0 +1,4 @@
+accessing
+attributes: aCollecion
```

Comment:
Typo in name of attribute. Maybe name it `aSet `or convert the collection to a set since we want a set don't we?